### DISCUSSION TITLE
Can ngrinder test websocket?

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#797] [Q&A]
Is there an example?  @junoyoon

**Comment by zhougy1989** [comment]
@junoyoon hello？

**Comment by matieqiang** [comment]
create groovy gradle script

**Comment by donggyu04** [comment]
@zhougy1989 

As mentioned above you can use the external library you want.
or create your own module for testing. refer to [Using nGrinder to perform load test for a socket.io app
](https://github.com/naver/ngrinder/wiki/Using-nGrinder-to-perform-load-test-for-a-socket.io-app) sample.
